test/db/abi/compilers/pelles_c

```diff
@@ -26,7 +26,7 @@ EXPECT=<<EOF
    38 * relocs
     4 * sections
     0 * segments
-  116 * strings
+  117 * strings
```

Comment:
Could you please paste here a difference?

Reply:
`+27  0x000075b1 0x004087b1 223 224  .rdata  ibm037  \°\j\k\l\m\n\o\p\q\r\ª\º\æ\¸\Æ\¤\µ\~\s\t\u\v\w\x\y\z\¡\¿\Ð\Ý\Þ\®\^\£\¥\·\©\§\¶\¼\½\¾\[\]\¯\¨\´\×\{\A\B\C\D\E\F\G\H\I\­\ô\ö\ò\ó\õ\}\J\K\L\M\N\O\P\Q\R\¹\û\ü\ù\ú\ÿ\\\÷\S\T\U\V\W\X\Y\Z\²\Ô\Ö\Ò\Ó\Õ\0\1\2\3\4\5\6\7\8\9\³\Û\Ü\Ù\Ú\`

`+0   0x00008638 0x0040a638 14  15   .data   ibm037  """"""""""""""`